src/services.py

```diff
@@ -677,9 +666,10 @@ def get_other_party_details(self, chat_room_id, user_id):
 
 
 class SocialLogin:
```

Comment:
Shall we name it `LinkedinLogin` to make it clearer? Then if we support more providers we can have different names.